src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom center;
+    animation: rotate-hand-minut 3600s infinite linear;
```

Comment:
There is a typo in the animation name `rotate-hand-minut`. It should likely be `rotate-hand-minute` to match the naming convention and avoid confusion.